msteams-platform/concepts/Cards/cards-reference.md

```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
 
 ## Cards supported in Teams
 
+### Adaptive card
+
+A customizable card that can contain any combination of text, speech, images, buttons, and input fields.
+
+#### Support
+
+| Bots in Teams | Messaging Extensions  | Connectors | Bot Framework |
```

Comment:
Are adaptive cards not supported in chats? We call out bots in teams, but nothing else.